{"uid":"629d8bab84a041ca","name":"test_get_categories","fullName":"test_progmusix.TestProgmusix#test_get_categories","historyId":"6b77a8b068594f06a7195162311b1f92","time":{"start":1725269095088,"stop":1725269095088,"duration":0},"status":"broken","statusMessage":"TypeError: TestProgmusix.test_get_categories() takes 0 positional arguments but 1 was given","statusTrace":"cls = <class '_pytest.runner.CallInfo'>\nfunc = <function call_and_report.<locals>.<lambda> at 0x7f81ec701360>\nwhen = 'call'\nreraise = (<class '_pytest.outcomes.Exit'>, <class 'KeyboardInterrupt'>)\n\n    @classmethod\n    def from_call(\n        cls,\n        func: Callable[[], TResult],\n        when: Literal[\"collect\", \"setup\", \"call\", \"teardown\"],\n        reraise: type[BaseException] | tuple[type[BaseException], ...] | None = None,\n    ) -> CallInfo[TResult]:\n        \"\"\"Call func, wrapping the result in a CallInfo.\n    \n        :param func:\n            The function to call. Called without arguments.\n        :type func: Callable[[], _pytest.runner.TResult]\n        :param when:\n            The phase in which the function is called.\n        :param reraise:\n            Exception or exceptions that shall propagate if raised by the\n            function, instead of being wrapped in the CallInfo.\n        \"\"\"\n        excinfo = None\n        start = timing.time()\n        precise_start = timing.perf_counter()\n        try:\n>           result: TResult | None = func()\n\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/runner.py:341: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/runner.py:242: in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy/_hooks.py:513: in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy/_manager.py:120: in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/threadexception.py:92: in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/threadexception.py:68: in thread_exception_runtest_hook\n    yield\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/unraisableexception.py:95: in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/unraisableexception.py:70: in unraisable_exception_runtest_hook\n    yield\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/logging.py:848: in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/logging.py:831: in _runtest_for\n    yield\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/capture.py:879: in pytest_runtest_call\n    return (yield)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/skipping.py:257: in pytest_runtest_call\n    return (yield)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/runner.py:174: in pytest_runtest_call\n    item.runtest()\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/python.py:1627: in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy/_hooks.py:513: in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy/_manager.py:120: in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\npyfuncitem = <Function test_get_categories>\n\n    @hookimpl(trylast=True)\n    def pytest_pyfunc_call(pyfuncitem: Function) -> object | None:\n        testfunction = pyfuncitem.obj\n        if is_async_function(testfunction):\n            async_warn_and_skip(pyfuncitem.nodeid)\n        funcargs = pyfuncitem.funcargs\n        testargs = {arg: funcargs[arg] for arg in pyfuncitem._fixtureinfo.argnames}\n>       result = testfunction(**testargs)\nE       TypeError: TestProgmusix.test_get_categories() takes 0 positional arguments but 1 was given\n\n/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/python.py:159: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestProgmusix","time":{"start":1725269094569,"stop":1725269095087,"duration":518},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_xunit_setup_method_fixture_TestProgmusix::0","time":{"start":1725269095220,"stop":1725269095329,"duration":109},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"suite","value":"test_progmusix"},{"name":"subSuite","value":"TestProgmusix"},{"name":"host","value":"fv-az1200-102"},{"name":"thread","value":"3169-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_progmusix"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"629d8bab84a041ca.json","parameterValues":[]}